cmd/main.go

```diff
@@ -58,11 +63,9 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	flag.StringVar(&secretNamespace, "webhook-secret-namespace", "llmaz-system", "The namespace of the secret that contains the webhook server's TLS certificate and key.")
```

Comment:
Let's rename this to `namespace` intuitively. Same to other places, secret is just one of the resources in the namespace I think.

Reply:
ok, i update this soon.